### DISCUSSION TITLE
pulsar version

### DISCUSSION THREAD

**Discussion by rere950303** [milvus-io/milvus#31806] [Q&A and General discussion]
I'm currently using milvus v2.3.12 version. For pulsar, is there any problem with using version 2.10.0 instead of 2.8.2?

**Comment by yiwangdr** [comment]
Since we only use the basic features of pulsar, it's unlikely going to have issues. 
However, it's untested. There could be unknowns. I'd suggest you to test & canary it before deploying it into production.
